.github/workflows/autotools-openmp.yml

```diff
@@ -0,0 +1,81 @@
+name: autotools-openmp
+# autotools on Ubuntu - run benchmark test. '--enable-openmp' no training tools
+on:
+  #push:
+  schedule:
+    - cron: 0 20 * * *
```

Comment:
Would it be sufficient to run this (and maybe some other actions, too) only on demand?

Most days there are no changes in the code, so running daily costs resources without any need.

Reply:
>Would it be sufficient to run this (and maybe some other actions, too) only on demand?

Yes, that's perfectly fine.

Reply:
See https://github.com/OCR-D/ocrd_all/blob/c28e3296c4b6321a1e86ea5b0be85767515d1256/.github/workflows/makeall.yml#L16 for an example.

Reply:
Thanks @stweil

Reply:
Note that the resource costs of daily CI actions can become really huge if those actions run also on hundreds (or even thousands) of forks. This can be reduced by limiting the daily actions to the primary repository only. Offering a manual trigger would still allow forks to run CI on their code, too.

Reply:
>This can be reduced by limiting the daily actions to the primary repository only. 

How do I do that?